source/codegen/metadata/nidcpower/functions.py

```diff
@@ -0,0 +1,2985 @@
+      }
+    ],
+    'returns': 'ViStatus'
+  },
+  'GetExtCalLastTemp': {
+    'parameters': [
+      {
+        'name': 'vi',
+        'direction': 'in',
+        'type': 'ViSession'
+      },
+      {
+        'name': 'temperature',
+        'direction': 'out',
+        'type': 'ViReal64'
+      }
+    ],
+    'returns': 'ViStatus'
+  },
+  'GetExtCalRecommendedInterval': {
+    'parameters': [
+      {
+        'name': 'vi',
+        'direction': 'in',
+        'type': 'ViSession'
+      },
+      {
+        'name': 'months',
+        'direction': 'out',
+        'type': 'ViInt32'
+      }
+    ],
+    'returns': 'ViStatus'
+  },
+  'GetNextCoercionRecord': {
+    'parameters': [
+      {
+        'name': 'vi',
+        'direction': 'in',
+        'type': 'ViSession'
+      },
+      {
+        'name': 'bufferSize',
+        'direction': 'in',
+        'type': 'ViInt32'
+      },
+      {
+        'name': 'coercionRecord',
+        'direction': 'out',
+        'type': 'ViChar[]',
+        'size': {
+          'mechanism': 'ivi-dance',
+          'value': 'bufferSize'
+        }
+      }
+    ],
+    'returns': 'ViStatus'
+  },
+  'GetNextInterchangeWarning': {
+    'parameters': [
+      {
+        'name': 'vi',
+        'direction': 'in',
+        'type': 'ViSession'
+      },
+      {
+        'name': 'bufferSize',
+        'direction': 'in',
+        'type': 'ViInt32'
+      },
+      {
+        'name': 'interchangeWarning',
+        'direction': 'out',
+        'type': 'ViChar[]',
+        'size': {
+          'mechanism': 'ivi-dance',
+          'value': 'bufferSize'
+        }
+      }
+    ],
+    'returns': 'ViStatus'
+  },
+  'GetSelfCalLastDateAndTime': {
+    'parameters': [
+      {
+        'name': 'vi',
+        'direction': 'in',
+        'type': 'ViSession'
+      },
+      {
+        'name': 'year',
+        'direction': 'out',
+        'type': 'ViInt32'
+      },
+      {
+        'name': 'month',
+        'direction': 'out',
+        'type': 'ViInt32'
+      },
+      {
+        'name': 'day',
+        'direction': 'out',
+        'type': 'ViInt32'
+      },
+      {
+        'name': 'hour',
+        'direction': 'out',
+        'type': 'ViInt32'
+      },
+      {
+        'name': 'minute',
+        'direction': 'out',
+        'type': 'ViInt32'
+      }
+    ],
+    'returns': 'ViStatus'
+  },
+  'GetSelfCalLastTemp': {
+    'parameters': [
+      {
+        'name': 'vi',
+        'direction': 'in',
+        'type': 'ViSession'
+      },
+      {
+        'name': 'temperature',
+        'direction': 'out',
+        'type': 'ViReal64'
+      }
+    ],
+    'returns': 'ViStatus'
+  },
+  'ImportAttributeConfigurationFile': {
+    'parameters': [
+      {
+        'name': 'vi',
+        'direction': 'in',
+        'type': 'ViSession'
+      },
+      {
+        'name': 'filePath',
+        'direction': 'in',
+        'type': 'ViConstString'
+      }
+    ],
+    'returns': 'ViStatus'
+  },
+  'InitExtCal': {
```

Comment:
Should this be marked as an `'init_method' : True,`? It seems like it initiates a ViSession.

Reply:
@reckenro, InitExtCal also initializes a session, hence I included 'init_method':True in the metadata. Thanks for putting this to notice.